connector_prestashop/__manifest__.py

```diff
@@ -15,7 +15,7 @@
         "product",
         "product_multi_category",  # oca/product-attribute
         "connector_ecommerce",  # oca/connector-ecommerce
-        "base_multi_image",  # oca/product-attribute
+        "product_multi_image",
```

Comment:
I think it's not useful to depend on product_multi_image in connector_prestashop module, it is needed mainly for export.

Reply:
Hi @bguillot,

Thanks you for your review.

`product_multi_image` allows to show imported images in product sheet, `base_multi_image' does not.

I think it makes no sense to import images if we don't show them.

Regards